content/en/blog/2025/announcing-etcd-3.6.md

```diff
@@ -0,0 +1,389 @@
+- **[Future roadmaps](#future-roadmaps)**
+
+## Introduction
+
+Today, we are releasing [etcd v3.6.0][]. It has been nearly four years since the release of etcd v3.5.0 on June 15, 2021,
+making the arrival of etcd v3.6.0 a significant milestone. This release introduces several new features, makes significant
+progress on long-standing efforts like downgrade support and migration to v3store, and addresses numerous critical & major
+issues. Notably, it also brings major optimizations in memory usage, improving efficiency and performance.
+
+Beyond being a CNCF graduated project, etcd has also become a Kubernetes SIG (sig-etcd), marking a new chapter in its
+evolution. The ecosystem is expanding as well—one notable example is the formation of the working group etcd-operator,
+which promises to greatly improve the usability and operability of etcd.
+
+In this blog, we will highlight the most significant changes introduced in etcd v3.6.0 and discuss the roadmap for future
+development. For a detailed list of changes, please refer to the [CHANGELOG-3.6][].
+
+A heartfelt thank you to all the contributors who made this release possible!
+
+## Security
+
+etcd takes security seriously and has implemented several measures to enhance it. We have integrated `govulncheck` to
+scan the source code and `trivy` to scan container images, both as part of our workflow checks. Additionally,
+we continue to follow the [Security Release Process][] to ensure vulnerabilities are properly managed and addressed.
+
+## Features
+
+### Migration to v3store
+The v2store has been deprecated since etcd v3.4 but could still be enabled via `--enable-v2`. It remained the source of
+truth for membership data. In etcd v3.6.0, v2store can no longer be enabled as the `--enable-v2` flag has been removed,
+and v3store has become the sole source of truth for membership data.
+
+While v2store still exists in v3.6.0, etcd will fail to start if it contains any data other than membership information.
+To assist with migration, etcd v3.5.18+ provides the `etcdutl check v2store` command to verify if v2store contains any
+non-membership data (see [PR 19113][]).
+
+The removal of v2store is still ongoing and is tracked in [issues/12913][].
+
+### Downgrade
+etcd v3.6.0 is the first version to fully support downgrade. The effort for this downgrade task spans
+both versions 3.5 and 3.6, and all related work is tracked in [issues/11716][].
+
+At a high level, the process involves migrating the data schema to the target version (e.g., v3.5)
+followed by a rolling downgrade.
+
+Ensure the cluster is healthy, and take a snapshot backup. Validate whether the downgrade is valid:
+```
+$ etcdctl downgrade validate 3.5
+```
+
+If the downgrade is valid, enable downgrade mode:
+```
+$ etcdctl downgrade anable 3.5
+```
+
+etcd will then migrate the data schema in the background. Once complete, proceed with the rolling downgrade.
+
+For details, refer to the [Downgrade-3.6] guide.
```

Comment:
Note: It links to the downgrade guide, which hasn't been merged yet. 
https://github.com/etcd-io/website/pull/965

Reply:
https://github.com/etcd-io/website/pull/965 has merged